src/textfield/_textfield.scss

```diff
@@ -29,7 +29,7 @@
   padding: $input-text-vertical-spacing 0;
 
   // Align buttons, if used.
-  & .mdl-button {
+  .mdl-button {
```

Comment:
Let's not remove the ampersand. It is more explicit which is what we want in the codebase.